frontend/src/features/projects/common/forms/AdditionalPropertyInformationForm.tsx

```diff
@@ -17,6 +17,17 @@ const AdditionalPropertyInformationForm = ({
   const formikProps = useFormikContext();
   return (
     <Container fluid>
+      <Form.Row>
+        <Form.Label column md={3}>
+          Project Approved On
+        </Form.Label>
+        <FastDatePicker
```

Comment:
Allow user to change the date that is automatically set when the project is approved.